src/precompiles/ArbWasm.sol

```diff
@@ -26,6 +26,14 @@ interface ArbWasm {
     // @return version the program version (0 for EVM contracts)
     function programVersion(address program) external view returns (uint16 version);
 
+    // @notice gets the uncompressed size of the program at the given address in bytes
+    // @return version the program version (0 for EVM contracts)
```

Comment:
I think this was accidentally copied over :)

Reply:
:+1: